src/super.c

```diff
@@ -165,6 +166,20 @@ static int scoutfs_sync_fs(struct super_block *sb, int wait)
 	return scoutfs_trans_sync(sb, wait);
 }
 
+/*
+ * Data dev is closed by generic code, but we have to explicitly close the meta
+ * dev.
+ */
+static void scoutfs_metadev_close(struct super_block *sb)
+{
+	struct scoutfs_sb_info *sbi = SCOUTFS_SB(sb);
+
+	if (sbi->meta_bdev) {
+		sync_blockdev(sbi->meta_bdev);
```

Comment:
We need to sync the bdev.  We don't use the blkdev page cache for metdata IO, and I bet it could be a problem (eventually) for forced unmount.  Drop that.   Then also set the sbi->meta_bdev to NULL so that this is safe to call multiple times?  We've had patterns like that in the past.